packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -973,23 +973,20 @@ rocksdb::Status ArbCore::reorgCheckpoints(
             std::move(std::get<std::unique_ptr<MachineThread>>(
                 nearest_machine_or_status));
 
-        if (initial_start) {
-            std::cerr << "Loading checkpoint, total gas used: "
-                      << nearest_machine->machine_state.output.arb_gas_used
-                      << ", L1 block: "
-                      << nearest_machine->machine_state.output.l1_block_number
-                      << ", L2 block: "
-                      << nearest_machine->machine_state.output.l2_block_number
-                      << ", log count: "
-                      << nearest_machine->machine_state.output.log_count
-                      << ", timestamp: "
-                      << std::put_time(
-                             std::localtime(
-                                 (time_t*)&nearest_machine->machine_state.output
-                                     .last_inbox_timestamp),
-                             "%c")
-                      << std::endl;
-        }
+        std::cerr << "Loading checkpoint, total gas used: "
+                  << nearest_machine->machine_state.output.arb_gas_used
+                  << ", L1 block: "
+                  << nearest_machine->machine_state.output.l1_block_number
+                  << ", L2 block: "
+                  << nearest_machine->machine_state.output.l2_block_number
+                  << ", log count: "
+                  << nearest_machine->machine_state.output.log_count
+                  << ", timestamp: "
+                  << std::put_time(
+                         std::localtime((time_t*)&nearest_machine->machine_state
+                                            .output.last_inbox_timestamp),
+                         "%c")
+                  << std::endl;
```

Comment:
Given the current issue, maybe we should put a messages read count in here? Other than that, LGTM

Reply:
excellent idea